fe/fe-core/src/main/java/org/apache/doris/nereids/rules/rewrite/MergeLimits.java

```diff
@@ -50,7 +50,7 @@ public Rule build() {
                             Math.min(upperLimit.getLimit(),
                                     Math.max(bottomLimit.getLimit() - upperLimit.getOffset(), 0)),
                             bottomLimit.getOffset() + upperLimit.getOffset(),
-                            upperLimit.getPhase(), bottomLimit.child()
+                            upperLimit.child().getPhase(), bottomLimit.child()
                     );
```

Comment:
why not check they have the same phase

Reply:
it is not necessary, because if they are same, use either bottom or upper is ok.